platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+
+##### Level 1, Provisional - Ad hoc
+
+Usage and satisfaction metrics are gathered in custom ways, if at all, for each platform and capability. Outcomes and measures of success are not consistently aligned across capabilities, and therefore corresponding insights are not gathered. User feedback and instrumentation of platform use may not be gathered, or if it is, it will be informal. Decisions are made based on anecdotal requirements and incomplete data.
+
+###### Characteristics:
+
+* No experience or opinions about how to measure success of platforms
+* Use familiar tools to gather common metrics with limited intent and forethought
+* Reliance on small amounts of quantitative data
+* Difficult to secure user participation - users believe their feedback isn't considered
+* If surveys are used, the questions change between runs, negating the ability to track progress
+
+###### Examples Scenarios:
+
+* A platform tech lead wants to improve the collaboration with users by adding key topics to their next quarterly planning. They decide to run a survey on what users would like to see. The response is overwhelming, which is exciting, but also results in a difficulty organizing and responding to all of the ideas. While some ideas influence the quarterly planning, the users do not see their ideas as being accepted and are less inclined to reply to the next survey.
+* The team wants to capture more data automatically, so they look for opportunities for easy collection such as test failures in CI. However not every team uses the same CI automation so the data is only available for Java applications even though some teams have moved on to writing their services in Scala.
+
+##### Level 2, Operationalized - Consistent collection
+
+Organization has an intentional goal to verify platform products meet the needs of their market of internal users. Actionable, structured collection of user feedback is valued. Dedicated teams or individuals might be assigned to gather feedback, ensuring a more consistent approach. Feedback channels, such as surveys or user forums, are standardized, and feedback is categorized and prioritized. Beyond user feedback, there is also an expectation that user experiences are instrumented to generate usage data over time.
+
+Challenges remain in translating feedback into actionable tasks. While there is a growing repository of user data, the organization might need help effectively understanding and integrating this feedback into a platform roadmap. It may be hard to ensure that users see tangible changes driven by their feedback.
+
+###### Characteristics:
+
+* Data collection is discussed as part of most major planning sessions or capability implementations
+* Users can track their feedback in relation to platform development and can see how their feedback is incorporated into the roadmap
```

Comment:
Didn't we say in 435 that ensuring feedback finds its way into the roadmap is hard at this level? This characteristic sounds contradictory to that.

Reply:
great catch. that was an evolution and it didn't make its way here!